Src/control.c

```diff
@@ -127,34 +115,34 @@ void PWM_ISR_CH2_Callback() {
 }
 
 // SysTick executes once each ms
-void PWM_SysTick_Callback() {
+void PWM_SysTick_Callback(void) {
   pwm_timeout++;
   // Stop after 500 ms without PPM signal
   if(pwm_timeout > 500) {
     //pwm_captured_ch1_value = 500;
```

Comment:
Why is this commented-out. Shouldn't we put also ch1 (Steering) to 500 which will results to cmd1 = 0, in case of Timeout?

Reply:
Yes, we should. I missed that one!